node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageReceiver.scala

```diff
@@ -202,9 +202,11 @@ class PeerMessageReceiver(
         //we don't want to have to request them manually
         sender.sendHeadersMessage()
         Future.successful(this)
-      case _: AddrMessage =>
+      case addr: AddrMessage =>
+        node.handlePeerGossipMessage(addr)
         Future.successful(this)
-      case _: AddrV2Message =>
+      case addr: AddrV2Message =>
+        node.handlePeerGossipMessage(addr)
         sender.sendSendAddrV2Message()
```

Comment:
I think this `sender.sendSendAddrV2Message()` is a mistake by me from awhile back, should be under the 
`case SendAddrV2Message => `

Reply:
This will allow us to receive addrv2 messages so we can test it too